CHANGELOG.rst

```diff
@@ -1,6 +1,38 @@
 Changelog
 =========
 
+0.8.1 (2018-12-20)
+------------------
+
+MLflow 0.8.1 introduces several improvements:
```

Comment:
Maybe "several major improvements" to indicate why these are called out above "Features" (which are presumably also improvements)

Reply:
hahaha, I just saw that you removed this word

Reply:
@aarondav @andrewmchen I've decided to go with *significant*. Hopefully these appeases both parties.